Character.cpp

```diff
@@ -90,25 +89,53 @@ int Character::takeDamage(int damage)
 
 
 //#include <assert>
+
+void defeatedOther(int attribute, std::unique_ptr<int>& initialAttribute)
```

Comment:
Pass by reference.  you're passing `attribute` by copy.  That means your class member isn't being updated
Also, dereference before you pass in the `initialAttribute`.  it looks a lot cleaner if you do that.
- [ ] revise

Reply:
i guess?

Reply:
you guess what?

Reply:
I guess this is done right now